_conferences/delphi-day-italian-conference-2024-padova.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Delphi Day Italian Conference - Padova 2024"
+website: https://www.delphiday.it/
+location: Padova, Italy
+online: false
+
+date_start: 2024-20-11
+date_end: 2024-20-11

```

Comment:
Those 2 dates were invalid and causing the build to fail @marcobreveglieri 
I've fixed it on main already

Reply:
I checked it visually several times but still couldn't see the problem, even though it was right in front of my eyes! :)

Thanks a lot for your support (and patience)! ;)